src/measure_theory/bochner_integration.lean

```diff
@@ -1666,4 +1670,98 @@ attribute [integral_simps] integral_neg integral_smul L1.integral_add L1.integra
 
 attribute [irreducible] integral L1.integral
 
+section integral_trim
+
+variables {H β γ : Type*} [normed_group H] [measurable_space H]
+  {m m0 : measurable_space β} {μ : measure β}
+
+/-- Simple function seen as simple function of a larger `measurable_space`. -/
```

Comment:
is there any way at all to avoid all the `@`s in this bit? it seems weird you need so many

Reply:
There are those two `measurable_space`, `m` and `m0`. The measurable space argument is implicit in almost every lemma in measure theory, and the one which will be guessed is `m0`. Every time I want to use a lemma for `m` instead, I need `@`.